mautic/mautic
# New campaign condition: Has click from channels

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | closes https://github.com/mautic/mautic/issues/6466
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Based on discussion  https://github.com/mautic/mautic/issues/6466#issuecomment-414236170

We can work with all clicked/non clicked contacts by new Campaign condition: Has click from channels 

![image](https://user-images.githubusercontent.com/462477/45871444-cd778280-bd8d-11e8-84e8-69e7a10835ba.png)

#### Steps to test this PR:
1.  Create clicks from channels (notification, email, sms..)
2.  Create campaign with new condition: Has click from channels  + test include/exclude URLs with * (see tooltip for more info)
3. Test If return true/false based on your contact state

Test by Mautibox: https://mautibox.com/6488/s/login 


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/6488"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
